docs/pages/setup/reference/metrics.mdx

```diff
@@ -109,6 +109,7 @@ Now you can see the monitoring information by visiting several endpoints:
 | `rx` | counter | Teleport | Number of bytes received. |
 | `server_interactive_sessions_total` | gauge | Teleport | Number of active sessions. |
 | `teleport_build_info` | gauge | Teleport | Provides build information of Teleport including gitref (git describe --long --tags), Go version, and Teleport version. The value of this gauge will always be 1. |
+| `teleport_registered_agents` | gauge | Teleport Auth | The number of Teleport agents with their version that have connected to the Teleport cluster. After disconnecting, a Teleport agent has a TTL of 10 minutes so this value will include agents that have recently disconnected but have not reached their TTL. |
```

Comment:
Not sure if it's relevant for the docs or not, but as I understand it Teleport has a problem in that when using Dynamo backends this takes much longer than the specified 10 minutes. Not sure if that'll occur in this metric or if the core team has figured out an alternate expiry than relying on the underlying database key expiry.

Reply:
Perhaps my understanding of this issue is incomplete but this checks against the expiration in the resource. This should mean that even if it is still present in DynamoDB/cache the resource still would have expired and would be ignored in the count. This was tested with a cluster using DynamoDB as the backend and could detect small differences with what `tctl` and `tsh` was giving me versus what prometheus was reporting (prometheus being more accurate). Essentially `tctl get proxies` returning current and old proxies but prometheus returning values that showed them being gone.